### ISSUE TITLE
Assess new WCAG 2.2 criteria and implement new accessibility checks if there are any gaps

### ISSUE THREAD

**Issue by edhamiltonHO** [UKHomeOffice/engineering-guidance-and-standards#306]
- [x] Review changes in WCAG 2.2
- [x] Ensure no focus items get hidden
- [ ] If any gaps raise new tickets to resolve
- [x] Make sure any accessibility checks or scripts are updated to reference 2.2 if necessary

**Comment by jeff-horton-ho-sas** [comment]
https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/

- [X] 2.4.11 Focus Not Obscured (Minimum) (AA)
    - We don't have banners/modals/other elements that appear over the page content
    - Updated layout accessibility guidance to indicate this assumption
- [X] 2.5.7 Dragging Movements (AA) 
    - The site is not interactive in this way - covered by existing assumption statements
- [ ] 2.5.8 Target Size (Minimum) (AA)
    - `axe-core` has an automated check for this.
    - The breadcrumbs are failing this on `/standards/managing-secrets/`, `/standards/signing-code-commits/`, and `/patterns/using-generative-ai/` - which looks to be an axe bug as they display the same as other pages that pass, and adding space doesn't help. 
    - Continuing to investigate this.
- [ ] 3.2.6 Consistent Help (A)
    - There is a feedback link in the phase banner which is in the same place on all pages  ✅ 
    - There is accessibility link in the footer that is consistent across all pages ✅ 
    - Question: do we want a more explicit get help link in the footer?
- [X] 3.3.7 Redundant Entry (A)
    - The site doesn't provide forms for users to submit
    - Updated layout accessibility guidance to indicate this assumption
- [X] 3.3.8 Accessible Authentication (Minimum) (AA)
    - The site does not have any authentication
    - Updated layout accessibility guidance to indicate this assumption
